kubelet/README.md

```diff
@@ -0,0 +1,32 @@
+# Kubelet Integration
+
+## Overview
+
+This integration gets Kubernetes metrics from kubelet
+
+* Visualize and monitor kubelet stats
+* Be notified about kubelet failovers and events.
+
+## Installation
+
+Install the `dd-check-kubelet` package manually or with your favorite configuration manager
+
+## Configuration
+
+Edit the `kubelet.yaml` file to point to your server and port, set tags to send along with metrics.
+
+## Validation
+
+When you run `datadog-agent info` you should see something like the following:
+
+    Checks
+    ======
+
+        kubelet
+        -----------
+          - instance #0 [OK]
+          - Collected 39 metrics, 0 events & 7 service checks
+
+## Compatibility
+
+The kubelet check is compatible with all major platforms where Kubernetes runs.
```

Comment:
I believe we should clarify the versions:
https://github.com/kubernetes/kubernetes/issues/32638#issuecomment-358250806.
it seems like the kubelet check only _fully_ works for 1.7.3+